lib/src/modules/task.cc

```diff
@@ -174,6 +180,13 @@ Task::Task(const fs::path& exec_prefix,
     client_.set_client_cert(crt, key);
     client_.set_supported_protocols(CURLPROTO_HTTPS);
     client_.set_proxy(proxy);
+
+#ifndef _WIN32
+    features_.insert("shell");
+#endif
+    if (!lth_exec::which("powershell").empty()) {
```

Comment:
Do powershell tasks work on non-windows systems? I'm unclear whether the "built-in task interpreter" for ps1 on windows does anything that would be necessary elsewhere. Specifically, it passes `-NoProfile -NonInteractive -NoLogo -ExecutionPolicy Bypass`.

Reply:
I'm not sure those arguments are needed with powershell on other platforms. They're present in the powershell 6 beta, but that might just be so scripts don't break.

Reply:
Hard-coded it so powershell is only matched on Windows.